prometheus-exporters/network-generic-ssh-exporter/templates/_config.yaml.tpl

```diff
@@ -17,6 +17,29 @@ lookup_sources:
       router_project: SELECT DISTINCT id, project_id FROM neutron.routers
 
 metrics:
+  zmq_rx_error:
+    regex: >-
+      ^ZmQ\:\s*RX\sCNT\:\s\d+\,\sBYTES\:\s\d+\,\sERRORS\:\s(\d*)
+    multi_value: true
+    value: $1
+    labels:
+      errors: $1  
```

Comment:
Vector label and value are assigned to the same match group, whenever this counter increases a new vector with a new label is created. That would just blow up the count of vectors.

Reply:
you are right